tests/unit_tests/test_rating.py

```diff
@@ -135,6 +137,16 @@ def test_initialization_transient(chained_ratings):
     assert chained_ratings["tmm_2v2"] == (300, 200)
 
 
+def test_initialization_transient_caching(chained_ratings):
+    # Return a new mock on each call. By default it will return the same mock
+    chained_ratings._get_initial_rating = mock.Mock(side_effect=mock.Mock)
+
+    chained_ratings["global"] = (1000, 50)
+
+    assert chained_ratings["tmm_2v2"] == chained_ratings["tmm_2v2"]
```

Comment:
Is this a typo? How will this ever fail?

Reply:
This checks that the rating is cached and not recomputed. If it were recomputed we would get different mock objects for the two calls.